stylesheets/card.css

```diff
@@ -36,66 +36,35 @@ body {
   padding: 0;
 }
 
+.img-box-text {
+  position: absolute;
+}
+
 .svgArrow {
   stroke-width: 1%;
   stroke: #BE5F4B;
   fill: transparent;
 
-  height: 100px;
+  height: 100%;
   width: 100%;
+  position: absolute;
 }
 
 .svgArrow.svgArrowWho {
-  margin-top: -80%;
+  margin-top: -58%;
   margin-left: -5%;
 }
 
 .svgArrow.svgArrowWhen {
-  margin-top: -30%;
-  margin-left: 22%;
+  margin-top: -27%;
```

Comment:
@aakside @anitasunshine - Wanted to confirm, is it August 29th? If not I'll need to adjust these values 😂

Reply:
Yup, it is